Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'cocoapods'
 gem 'cocoapods-update-if-you-dare', '~> 0.1.1'
+gem 'colored'
```

Comment:
We actually do, since `cocoapods-update-if-you-dare` does not specify this dependency explicitly. And `cocoapods` moved to `colored2`: https://github.com/CocoaPods/Xcodeproj/pull/463

I opened a PR on the project itself to fix it: https://github.com/Ashton-W/cocoapods-update-if-you-dare/pull/2